From 22af960311a1d6b3979f5a34e093464cdd36ff21 Mon Sep 17 00:00:00 2001
From: uazo <uazo@users.noreply.github.com>
Date: Tue, 11 Jan 2022 13:59:32 +0000
Subject: [PATCH 003/177] Do not link with libatomic

Compiler has built-in support thus libatomic linking is no more needed
This patch allows building with use_sysroot=false
---
 base/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/BUILD.gn b/base/BUILD.gn
index dbef16bc35..1ba8f0d564 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -1464,7 +1464,7 @@ mixed_component("base") {
   if (!use_sysroot &&
       (is_android || ((is_linux || is_chromeos) && !is_chromecast)) &&
       host_toolchain != "//build/toolchain/cros:host") {
-    libs += [ "atomic" ]
+    #libs += [ "atomic" ]
   }
 
   if (use_allocator_shim) {
-- 
2.35.1

